eng/packages/http-client-csharp/generator/TestProjects/Local/Basic-TypeSpec/src/BasicTypeSpec.csproj

```diff
@@ -12,5 +12,6 @@
   <ItemGroup>
     <PackageReference Include="System.ClientModel" Version="1.1.0-beta.4" />
```

Comment:
I saw you deleted a lot of using for ClientModel. Then do we still need this reference?

Reply:
because not everything has been abstracted, such as `ApiKeyCredential` will be handled later, then we can remove this.